### DISCUSSION TITLE
VR Server on Windows

### DISCUSSION THREAD

**Discussion by Walishaha** [Velocidex/velociraptor#2884] [Q&A]
Hi Mike, 

I am looking for a deployment scenario where I could install the VR server on a Windows machine and then clients with a mix of Linux and Windows. I can not see some documentation that explains how to generate the server package for Windows.

I want this to be persistent. 

Can you please guide me? I was able to make the deployment where the server is running on Ubuntu and then clients on Windows and Linux but not where I could install the server on Windows.

**Comment by scudette** [comment]
This is not a supported configuration for production as the io performance on windows is not high and we don't test this configuration on loaded environments.

Having said that, some people are still running it that way. You can install the service using velociraptor --config server.config.yam server_service install

Or you can create your own service with SC.exe 

It's ok to run like this for toy deployments and we do that all the time in training etc. So it should functionality work but I don't know how it will do under load.

**Comment by Walishaha** [comment]
Thank you, Mike. It worked for me.
